diff -up Python-3.4.2/Lib/test/test_exceptions.py.ppc Python-3.4.2/Lib/test/test_exceptions.py
--- Python-3.4.2/Lib/test/test_exceptions.py.ppc	2014-10-08 04:18:13.000000000 -0400
+++ Python-3.4.2/Lib/test/test_exceptions.py	2015-01-12 05:56:15.922833055 -0500
@@ -429,6 +429,7 @@ class ExceptionTests(unittest.TestCase):
         self.assertIsNone(e.__context__)
         self.assertIsNone(e.__cause__)
 
+    @unittest.skip('rhbz#846849')
     def testChainingDescriptors(self):
         try:
             raise Exception()
diff -up Python-3.4.2/Lib/test/test_io.py.ppc Python-3.4.2/Lib/test/test_io.py
--- Python-3.4.2/Lib/test/test_io.py.ppc	2015-01-12 05:56:15.922833055 -0500
+++ Python-3.4.2/Lib/test/test_io.py	2015-01-12 05:58:23.482833055 -0500
@@ -3296,12 +3296,15 @@ class SignalsTest(unittest.TestCase):
                 if e.errno != errno.EBADF:
                     raise
 
+    @unittest.skip('rhbz#846849')
     def test_interrupted_write_unbuffered(self):
         self.check_interrupted_write(b"xy", b"xy", mode="wb", buffering=0)
 
+    @unittest.skip('rhbz#846849')
     def test_interrupted_write_buffered(self):
         self.check_interrupted_write(b"xy", b"xy", mode="wb")
 
+    @unittest.skip('rhbz#846849')
     # Issue #22331: The test hangs on FreeBSD 7.2
     @support.requires_freebsd_version(8)
     def test_interrupted_write_text(self):
diff -up Python-3.4.2/Lib/test/test_raise.py.ppc Python-3.4.2/Lib/test/test_raise.py
--- Python-3.4.2/Lib/test/test_raise.py.ppc	2014-10-08 04:18:14.000000000 -0400
+++ Python-3.4.2/Lib/test/test_raise.py	2015-01-12 05:56:15.922833055 -0500
@@ -151,6 +151,7 @@ class TestRaise(unittest.TestCase):
 
 class TestCause(unittest.TestCase):
 
+    @unittest.skip('rhbz#846849')
     def testCauseSyntax(self):
         try:
             try:
diff -up Python-3.4.2/Lib/test/test_traceback.py.ppc Python-3.4.2/Lib/test/test_traceback.py
--- Python-3.4.2/Lib/test/test_traceback.py.ppc	2014-10-08 04:18:14.000000000 -0400
+++ Python-3.4.2/Lib/test/test_traceback.py	2015-01-12 05:56:15.922833055 -0500
@@ -314,6 +314,7 @@ class BaseExceptionReportingTests:
         self.check_zero_div(blocks[0])
         self.assertIn('inner_raise() # Marker', blocks[2])
 
+    @unittest.skip('rhbz#846849')
     def test_context_suppression(self):
         try:
             try:
